CSS/style.css

```diff
@@ -631,3 +714,303 @@ section.contact_me_mobile > hr {
+  font-family: 'Poppins', system-ui, Courier, monospace;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 60px;
+  margin-bottom: 1rem;
+  color: var(--h1-color);
+}
+
+.desktop-landing-content .wrapper {
+  max-width: 15ch;
+  white-space: nowrap;
+}
+
+.desktop-landing-content .lets-con-desk-txt {
+  font-size: 16px;
+  font-family: 'Poppins', sans-serif, Courier, monospace;
+  min-width: 117px;
+  max-height: 24px;
+  color: #7f8cff;
+  text-align: left;
+  margin-left: 12rem;
+  margin-bottom: 0.5rem;
+}
+
+.desktop-landing-content .lets-con-desk-icon {
+  width: 11rem;
+  color: #7f8cff;
+  text-align: left;
+  margin-left: 12rem;
+}
+
+.desktop-landing-content .lets-con-desk-icon ul {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 170px;
+  list-style: none;
+}
+
+.desktop-card-section {
+  display: grid;
+  background-color: #f7f7f9;
+  width: 100%;
+  height: 100%;
+  margin-top: 50px;
+}
+
+.desktop-card {
+  display: flex;
+  border-radius: 24px;
+  justify-self: center;
+  margin-bottom: 5rem;
+  margin-top: 5rem;
+  border: 1px solid #dfe1e6;
+  background-color: #fff;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  height: 496px;
+  gap: 1rem;
+}
+
+.desk_main_content {
+  margin-left: 10px;
+  margin-right: 10px;
+  margin-top: 1rem;
+}
+
+.desk_main_content h1 {
+  font-size: 40px;
+  font-weight: bold;
+  font-family: 'Poppins', system-ui, Courier, monospace;
+  color: #172b4d;
+  text-align: left;
+  margin-top: 1rem;
+}
+
+.desktop-section .desk_main_content p {
+  display: flex;
+  align-items: center;
+  font-size: 16px;
+  font-style: normal;
+  font-family: 'Poppins', sans-serif, Courier, monospace;
+  width: 496px;
+  height: 50px;
+  color: #344563;
+  text-align: left;
+  margin-top: 2rem;
+  margin-left: 1px;
+}
+
+.desk_main_img1 {
+  display: flex;
+  background-image: url('../Images/deskcard1.svg');
+  background-repeat: no-repeat;
+  margin-top: 10px;
+  width: 544px;
+  height: 448px;
+  border-radius: 8px;
+  margin-right: 10px;
+}
+
+.desk_main_img2 {
+  display: flex;
+  background-image: url('../Images/deskcard1.svg');
+  background-repeat: no-repeat;
+  margin-top: 10px;
+  width: 544px;
+  height: 448px;
+  border-radius: 8px;
+  margin-right: 10px;
+}
+
+.desk_main_img3 {
+  display: flex;
+  background-image: url('../Images/deskcard1.svg');
+  background-repeat: no-repeat;
+  margin-top: 10px;
+  width: 544px;
+  height: 448px;
+  border-radius: 8px;
+  margin-right: 10px;
+}
+
+.desk_main_img4 {
+  display: flex;
+  background-image: url('../Images/deskcard1.svg');
+  background-repeat: no-repeat;
+  margin-top: 10px;
+  width: 544px;
+  height: 448px;
+  border-radius: 8px;
+  margin-right: 10px;
+}
+
+section.desktop_ams {
+  display: flex;
+  width: 100%;
+  border: 1px solid #dfe1e6;
+  background-color: #fff;
+  padding-top: 5rem;
+  padding-bottom: 10rem;
+  padding-left: 5rem;
+  border-top-right-radius: 150px;
+}
```

Comment:
### Sorry this wasn't mentioned before so I set it as [OPTIONAL] changes, still its highly recommended since the website should be responsive in all different screen sizes 💪🏼

While you improved the about section layout 👏🏼 as requested by the previous reviewer there are some issues that you need to work on, currently, the content is overlapping at some screen sizes as shown in the below screenshot.

![image](https://user-images.githubusercontent.com/68030297/229243710-1380e26d-f6e9-446d-8e65-cd4b53f9f68d.png)

- You can set a `media-query` at the screen size where the issue starts accused and change the property of `flex-direction` from `row` to `column` to the `desktop_ams` class so the section looks the same as in the mobile version from the Figma Design.

![Screenshot from 2023-03-31 23-29-23](https://user-images.githubusercontent.com/68030297/229243924-2ef54224-a885-4064-a98e-8474e4f47f1e.png)